### PR TITLE
Fix failing tests in master

### PR DESCRIPTION
**Pull request by havogt**
- test_grid was not updated in #1444
- test_storage_facility needs a CUDA compiler for storage_cuda

Sorry for that.

### CODE REVIEW

File: tests/unit_tests/storage/test_storage_facility.cu [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+ * GridTools
+ *
+ * Copyright (c) 2014-2019, ETH Zurich
+ * All rights reserved.
+ *
+ * Please, refer to the LICENSE file in the root directory.
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#include "test_storage_facility.cpp"
```
Comment (havogt) [inline tests/unit_tests/storage/test_storage_facility.cu]: I didn't have a better idea... Let me know if you have one.

**Comment by havogt** [comment]
launch jenkins

File: tests/unit_tests/storage/test_storage_facility.cu (hunk 1 above)
Comment (anstaf) [reply tests/unit_tests/storage/test_storage_facility.cu]: I think that  you don't need dummy `.cu` file here. `gridtools_add_unit_test` will do normalization

Comment (havogt) [reply tests/unit_tests/storage/test_storage_facility.cu]: I tried it, but it didn't work. I can try again, because maybe I got lost in the options...

Comment (havogt) [reply tests/unit_tests/storage/test_storage_facility.cu]: You are right, I got lost in the dependencies...

**Comment by havogt** [comment]
launch jenkins

**Comment by havogt** [comment]
launch jenkins

**Comment by havogt** [comment]
launch jenkins

**Pull request merged**